netty/src/main/java/io/grpc/netty/NettyServerStream.java

```diff
@@ -53,6 +58,13 @@
   private final NettyServerHandler handler;
   private final Http2Stream http2Stream;
   private final WriteQueue writeQueue;
+  private final Attributes attributes;
+
+  private static Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
+      Attributes.Key.named("remote-addr");
```

Comment:
So we can bike-shed on names. To make security people happy let's just throw 'insecure-' or similar in front of this name.